### PR TITLE
Fix browserify tests

### PR DESCRIPTION
**Pull request by viddo**
Turns out the “_” prefix actually makes it skip that bundle. Removed the prefix and updated references to this file to fix it.

### CODE REVIEW

File: lib/build/tasks/uglify.js [hunk 1]
```diff
@@ -28,7 +28,7 @@ exports.task = function() {
 
   var defaultOptions = {
     sourceMap: true,
-    // banner: '/*! v<%= pkg.version %> - <%= grunt.template.today("yyyy-mm-dd") %> */'
+    banner: '/*! v<%= pkg.version %> - <%= grunt.template.today("yyyy-mm-dd") %> */'
```
Comment (viddo) [inline lib/build/tasks/uglify.js]: Also saw that the banner was accidentally removed in the previous PR related to enabling source maps. :(

**Comment by viddo** [comment]
@xavijam related to yesterday's changes. Await clinker and deploying on staging to verify build also still working.

File: lib/build/tasks/uglify.js (hunk 1 above)
Comment (xavijam) [inline lib/build/tasks/uglify.js]: no prob!

**Comment by xavijam** [comment]
ok! :jp:

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/3616/))

**Pull request merged**